### PR TITLE
docs(ES-68): update readme, remove unnecessary todo

### PR DESCRIPTION
**Pull request by eliasmueller**

### CODE REVIEW

**Review by MarcelWolkober — APPROVED** [review]
Nice ReadMe :)

File: README.md [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
-## Build
-```shell
-./mvnw package
-```
+The plugin only works (without adaptions) in the context of the entire DeMAF application using the [deployment-config](https://github.com/UST-DeMAF/deployment-config).
+The documentation for setting up the entire DeMAF application locally is [here](https://github.com/UST-DeMAF/EnPro-Documentation).
+
+## Build and Run Application
+
+You can run the application without the [deployment-config](https://github.com/UST-DeMAF/deployment-config) but it will not run as it needs to register itself at the [analysis-manager](https://github.com/UST-DeMAF/analysis-manager).
```
Comment (heidrifx) [inline README.md:10]: I'm a bit confused with the "register at". Wouldn't it be "register in" or "register with"? I could be wrong, of course.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,47 @@
...
+
+If you want to boot it locally nevertheless use the following commands.
 
-## Run
 ```shell
 ./mvnw spring-boot:run
 ```
-or (requires build):
+or:
```
Comment (heidrifx) [inline README.md:17]: Maybe add "[...] to use the built package"?

File: README.md [hunk 3]
```diff
@@ -1,22 +1,47 @@
...
 git submodule update --remote
-```
\ No newline at end of file
+```
+
+## Ansible-Specific Configurations
+This plugin has some minor specialities compared to other DeMAF transformation plugins:
+1. No directories: Ansible projects are created in a folder structure, where it is possible to have multiple overlapping main files.
+Therefore, it is not possible to scan directories, but always only one main file.
+Of course, matching subdirectories, e.g. for roles are possible to read.
```
Comment (heidrifx) [inline README.md:40]: Maybe add this to the TODO.md?

**Review by heidrifx — COMMENTED** [review]
Nice README! Just some small things from me that could be resolved within minutes, then we can merge.

File: README.md (hunk 1 above)
Comment (eliasmueller) [reply README.md:10]: https://grammarhow.com/register-in-or-register-at/ I think it is okay :D

File: README.md (hunk 3 above)
Comment (eliasmueller) [reply README.md:40]: this is not a todo, but actually by design, I already discussed it with Marcel :)

File: README.md (hunk 2 above)
Comment (eliasmueller) [reply README.md:17]: done

**Pull request merged**